packages/wallets/src/features/cfd/screens/Jurisdiction/jurisdiction-contents/jurisdiction-bvi-contents.ts

```diff
@@ -16,7 +16,8 @@ export const getJurisdictionBviContents = (): TJurisdictionCardItems => ({
             {
                 clickableDescription: [
                     {
-                        text: 'Dynamic Leverage', // onClick: toggleDynamicLeverage,
+                        tag: 'dynamic-leverage',
```

Comment:
nitpick cuz we using camelcase for all stuff:
```suggestion
                        tag: 'dynamicLeverage',
```

Reply:
Updated.